src/ontology/cl-edit.owl

```diff
@@ -2728,6 +2728,8 @@ Declaration(ObjectProperty(obo:RO_0002202))
 Declaration(ObjectProperty(obo:RO_0002215))
 Declaration(ObjectProperty(obo:RO_0002292))
 Declaration(ObjectProperty(obo:RO_0013001))
+Declaration(ObjectProperty(obo:RO_0015015))
+Declaration(ObjectProperty(obo:RO_0015016))
```

Comment:
@dosumis @matentzn do OPs have to be declared? I thought imports didnt need to be declared? Is it different for OPs? It's not a huge issue just curious.

Reply:
Dont worry about it. Redundant declarations are common and not harmful. To answer your question: they do not have to be declared, if they are declared in the imported ontology!